server/main.lua

```diff
@@ -44,9 +44,20 @@ end)
 lib.addCommand('startshow', {
```

Comment:
Should we expose this function as an export as well?

Reply:
possibly, added export